Libraries/PeriphDrivers/Source/WUT/wut_reva_me14.svd

```diff
@@ -45,7 +45,21 @@
         </fields>
       </register>
       <register>
-        <name>INTR</name>
+        <name>PWM</name>
```

Comment:
The WUT only supports one-shot, continuous, and compare modes.  Many of these additional fields are not available.

Reply:
Thanks. I believe all the parts supporting WUT still contain the PWM-related fields in their register files. I'll go ahead and update those files.